src/app/containers/manuscript/manuscript-toolbar.tsx

```diff
@@ -135,8 +145,8 @@ export const ManuscriptToolbar: React.FC<ManuscriptToolbarProps> = (props) => {
             { title: 'Heading 3', enabled: canToggleHeading(3), action: toggleHeading(3) },
             { title: 'Heading 4', enabled: canToggleHeading(4), action: toggleHeading(4) },
             { title: 'Paragraph', enabled: canToggleParagraph, action: toggleParagraph },
-            { title: 'Bulleted List', enabled: false, action: undefined },
-            { title: 'Numbered List', enabled: false, action: undefined },
+            { title: 'Bulleted List', enabled: canToggleList('bullet'), action: toggleBulletList },
```

Comment:
I noticed that the heading controls are still enabled when in a list item, but selecting them doesn't do anything. Not sure what the expected behaviour should be, I suspect that it should remain a list but the text styled as a heading. But it could also be perfectly valid to have it remove the list style and make a heading, or to disable the control.
Also paragraph is disabled, might be nice to have it enabled to clear the list styling. At present, the only way I can delete a list is using backspace/delete on the keyboard which I don't think is ideal.